index.css

```
@@ -5,25 +5,37 @@ body {
 }
 
 .banner {
-	background-color: #000000;
+	background-color: #000;
 }
 
-.aligned {
-	padding-left: 20%;
+nav div {
+	text-align: center;
+}
+
+.logo {
+	max-width: 80%;
+	height: auto;
+}
+
+.menu-child {
+	display: inline-flex;
 }
 
 .menu {
+	margin-top: -5px;
 	background-color: #101010;
-	display: flex;
+	text-align: center;
 }
 
 .menu .button {
 	color: #E03FB1;
 	border-left: 1px solid #232323;
 	text-align: center;
 	cursor: pointer;
-	font: bold 1.5rem Trebuchet MS,Verdana,sans-serif;
-	padding: 15px 25px 15px 25px;
+	font-weight: bold;
+	font-size: 1rem;
+	font-family: Trebuchet MS, Verdana, sans-serif;
+	padding: .7em 1.3em .7em 1.3em;
```

I'm trying to keep the decimals with the noob 0 for readability
```suggestion
	padding: 0.7em 1.3em 0.7em 1.3em;
```